echo-pubsub-google/src/main/java/com/netflix/spinnaker/echo/pubsub/google/GooglePubsubSubscriber.java

```diff
@@ -187,8 +187,8 @@ public void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
         .messagePayload(messagePayload)
         .messageAttributes(messageAttributes)
         .pubsubSystem(pubsubSystem)
-        .ackDeadlineMillis(5 * TimeUnit.SECONDS.toMillis(ackDeadlineSeconds)) // Set a high upper bound on message processing time.
-        .retentionDeadlineMillis(TimeUnit.DAYS.toMillis(7)) // Expire key after max retention time, which is 7 days.
+        .ackDeadlineSeconds(5 * ackDeadlineSeconds) // Set a high upper bound on message processing time.
+        .retentionDeadlineSeconds(7 * 24 * 60 * 60) // Expire key after max retention time, which is 7 days.
```

Comment:
Why not `DAYS.toSeconds()`?

Reply:
Good questions. Because that returns a long, and instead of converting the value I just want to use math. The "7 days" comment next to it should explain enough.

Reply:
SGTM.